MapboxDirections/MBDirectionRoute.swift

```diff
@@ -1,60 +1,20 @@
 import Polyline
 
 /**
- A `Route` object defines a single route that the user can follow to visit a series of waypoints in order. The route object includes information about the route, such as its distance and expected travel time. Depending on the criteria used to calculate the route, the route object may also include detailed turn-by-turn instructions.
- 
- Typically, you do not create instances of this class directly. Instead, you receive route objects when you request directions using the `Directions.calculate(_:completionHandler:)` method. However, if you use the `Directions.url(forCalculating:)` method instead, you can pass the results of the HTTP request into this class’s initializer.
```

Comment:
Something along these lines is still necessary, so that developers (especially when coding in Objective-C) don’t mistakenly create one of these objects directly.

/ref https://github.com/mapbox/MapboxDirections.swift/pull/236#discussion_r172036593